docs/concepts.rst

```diff
@@ -842,61 +842,135 @@ values by "case" instead of by entity.
     # Returns `{'black cat', 'brown cat', 'brown fox'}`.
     builder.build().get('colored_animal', 'set')
 
-Other Features
---------------
+Parallel and Cloud Execution
+----------------------------
+
+By default, Bionic computes values one at a time. Requesting an entity value
+with :meth:`Flow.get <bionic.Flow.get>` can lead to a long computation, as
+Bionic may need to compute that entity's dependencies, and their dependencies,
+and so on.
+
+Bionic can take advantage of more computing resources by using local
+multiprocessing or Google AI Platform. However, this feature is not enabled
+by default because there is an overhead in starting up external processes, the
+structure of the graph affects performance improvement, and there are
+serialization requirements for the entity functions. See :ref:`caveats section <parallel-caveats>`
```

Comment:
You at least need a "the" before "caveats section". But you can also just "See Caveats for more details."